pyftpdlib/handlers.py

```diff
@@ -3449,6 +3453,15 @@ def __init__(self, conn, server, ioloop=None):
 
         def __repr__(self):
             return FTPHandler.__repr__(self)
+            
+        #commented out prints - for different Python version support
```

Comment:
`flake8` (which is run by the Travis builds) is complaining about this line (and the others below) not having a space after the `#` character